omod/src/main/java/org/openmrs/module/spa/SpaActivator.java

```diff
@@ -11,19 +11,57 @@
 
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
+import org.openmrs.api.AdministrationService;
+import org.openmrs.api.context.Context;
 import org.openmrs.module.BaseModuleActivator;
+import org.openmrs.module.ModuleConstants;
+import org.openmrs.module.ModuleException;
+import org.openmrs.util.OpenmrsUtil;
+
+import java.io.File;
 
 public class SpaActivator extends BaseModuleActivator {
 
 	private final Log log = LogFactory.getLog(getClass());
+	private final String FRONTENT_DIRECTORY = "frontend";
```

Comment:
Not a big deal though, how about replacing 'frontend' with 'spa'? 😊

Reply:
This is implementer-facing. For implementers, "frontend" does what it says on the tin. There's no reason to expect implementers to understand what "spa" is. Initializer calls their directory "configuration," not "initializer." And no other module is presently trying to use an app data dir called "frontend."

Reply:
@dkayiwa what is your response?

Reply:
Like i said that it is not a big deal, we can just go with what you have. 😊

Reply:
Thanks.